pyvista/plotting/plotting.py

```diff
@@ -3758,17 +3758,12 @@ def update_scalar_bar_range(self, clim, name=None):
             self.mapper.scalar_range = clim
             return
 
-        # Use the name to find the desired actor
-        def update_mapper(mapper_helper):
-            mapper_helper.scalar_range = clim
-            return
```

Comment:
This helper function is one line and used in one place, so I've inlined its contents in the loop.